### ISSUE TITLE
Feature request: Add example of "extra download" certificate in chain

### ISSUE THREAD

**Issue by balazsgerlei** [lgarron/badssl.com#538]
Unfortunately I cannot share an example but I worked with an issue in an Android app where the WebView component refuses to load a page that has a certificate in it's chain that is marked as "extra download" by a tool like SSL Labs' SSL Server Test (https://www.ssllabs.com/ssltest/index.html).

I'm not an expert on certificates so I'm not sure what this issue exactly mean, but all of the browsers I tried on Android loaded such site, but WebView refused to do so. Exactly because the server I experienced this issue cannot be shared and also would probably changed soon to fix the issue, it would be very useful to have a test/demo page that exhibits this issue. Can someone help me with this, I think it would be a great addition to badssl.com

Thank you in advance!

**Comment by bwbroersma** [comment]
I think this is already covered with `incomplete-chain.badssl.com`, see: 
https://www.ssllabs.com/ssltest/analyze.html?d=incomplete%2dchain.badssl.com

It will list:
- Chain issues: Incomplete
- Extra download (for the intermediate certificate)

However, currently it's expired:
- #526

**Comment by balazsgerlei** [comment]
Thanks! I will check it out, for some reason I could not reproduce the issue I found with any of the demo but maybe I was wrong, I will double check!

Or maybe it was the expired cert that gave a different result, I'll also check once that has been fixed!

**Comment by balazsgerlei** [comment]
Hi!

Thanks for the patience and the suggestion @bwbroersma, the other issue with the certificate has been resolved and I finally managed to try and indeed, with the correctly working `incomplete-chain.badssl.com` I could reproduce the issue I encountered in production so I can finally debug the component that I encountered an issue with in this scenario and possibly submit an issue for it's developers!

Thanks a bunch!
